repository/SmalltalkCI-Core.package/SCITestRunner.class/class/runClasses.spec..st

```diff
@@ -6,5 +6,5 @@ runClasses: aCollectionOfClasses spec: aSCISpec
     select: [ :each | (each includesBehavior: TestCase) and: [ each isAbstract not ] ])
       asSortedCollection: [ :a :b | a name <= b name ].
   classes
-    do: [ :each | each addToSuiteFromSelectors: suite ].
-  ^ self runSuite: suite spec: aSCISpec
\ No newline at end of file
+    do: [ :each | suite addTest: each buildSuite ].
```

Comment:
Hang on... why does this work? This adds a suite to a collection of tests, right?